server/scripts/db.init.js

```diff
@@ -0,0 +1,188 @@
+const mongoose = require('mongoose');
+const path = require('path');
+const models = require('../models');
+require('dotenv').config({ path: path.join(__dirname, '../../.env') });
+require('dotenv').config({ path: path.join(__dirname, '../../.env.sample') });
```

Comment:
@IgorChaikin Its not required to read `.env.sample` config. `.env.sample` - its just example, which allows to detect which environment variables are used in project